app/Providers/AppServiceProvider.php

```diff
@@ -15,6 +18,22 @@ class AppServiceProvider extends ServiceProvider
     public function boot()
     {
         Schema::defaultStringLength(191);
+
+        VerifyEmail::toMailUsing(function ($notifiable) {
+            $verifyUrl = URL::temporarySignedRoute(
+                'home.email.verify',
+                now()->addMinutes(60),
+                ['id' => $notifiable->getKey()]
+            );
+
+            return (new MailMessage())
+                ->subject( __('email.subject'))
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: PSR2.Methods.FunctionCallSignature.SpaceAfterOpenBracket</br>**Severity**: error</br>**File**: app/Providers/AppServiceProvider.php</br>**Line**: 30</br>Space after opening parenthesis of function call prohibited